### DISCUSSION TITLE
Como cambio de idioma / How to change language

### DISCUSSION THREAD

**Discussion by marcelo558** [Courseplay/Courseplay_FS22#3274] [Q&A]
Buenos dias, alguien me indicaría cómo cambiar el idioma al español ya que no lo consigo cambiar Gracias

Good morning, could someone tell me how to change the language to Spanish since I can't change it. Thank you.

**Comment by Tensuko** [comment]
You got to change the game language to do that.
For Steam you can right click your game and somehwere in the menu is the option to change it.
For Giants Version and other plarforms I don't know.

**Comment by marcelo558** [comment]
Necesito cambiar el idioma de Courseplay nada más, el juego ya lo tengo en español

**Comment by Leanquiroga** [comment]
A mi también me pasaba lo mismo. Por lo visto si tienes el juego en "spanish LA", el mod no lo reconoce. Tendrás que cambiar el idioma del juego a "spanish", aplicar los cambios y entrar a la partida para confirmar. Suerte.

**Reply by Leanquiroga** [reply]
PD: Tengo la version 7.4.0.0 del mod
